Add steps on unattended upgrades in load balanced setups (#3064)
umbraco/UmbracoDocs: Getting-Started/Setup/Upgrading/general.md

The upgrade is run after Composers but before Components. This is because the migration requires services that are registered in Composers and Components requires that Umbraco and the database is ready.
:::

### Unattended upgrades in a load balanced setup

Follow the steps outlined below to use run unattended upgrades in a load balanced setup.

1. Upgrade Umbraco via NuGet in Visual Studio. Make sure the `Umbraco.Core.ConfigurationStatus` key in `appSetting` in the `web.config` file is updated to match the **target version**.

> 2. Deploy to all environments - be sure to include the `appSetting`.

Reviewer: ```suggestion
2. Deploy to all environments, including the updated `appSetting` for `Umbraco.Core.ConfigurationStatus`.
```

> 3. Set the `Umbraco.Core.RuntimeState.UpgradeUnattended` key in `appSetting` in the `web.config` to `true` for **only the Main server**.

Reviewer: ```suggestion
3. Set the `Umbraco.Core.RuntimeState.UpgradeUnattended` key in `appSetting` in the `web.config` to `true` for **the Main server only**.
```
